README.md

```diff
@@ -1,2 +1,313 @@
+The only difference occurs when creating an apartment migration file using the command:
+ 
+```
+php artisan apartment:migration [name of migration]
+```
+
+which will create a file within your migrations folder. In this example we've created a migration called CreateProductsTable 
+which created the file: 2017_05_17_203315_CreateProductsTable.php
+
+```
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use BuildEmpire\Apartment\ApartmentMigration;
+
+/**
+ * Apartment migration file.
+ *
+ * You must set the table name with the table property, and place it anywhere you reference the table in your schemas.
+ */
+class CreateProductsTable extends ApartmentMigration
+{
+    /**
+    * You must use this table variable in your schema migrations as the table name.
+    */
+    protected $table = '';
+
+    /**
+     * Run the migrations.
+     *
+     * @return  void
+     */
+    public function apartmentUp()
+    {
+        Schema::create($this->table, function (Blueprint $table) {
+            $table->increments('id');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return  void
+     */
+    public function apartmentDown()
+    {
+        Schema::drop($this->table);
+    }
+}
+```
+The important things to notice in the apartment migration file are these two areas:
+
+```
+protected $table = '';
+```
+and the schema calls
+```
+Schema::create($this->table, functi....
+Schema::drop($this->table);
+```
+
+
+You must specify the table name that you wish to update, create or any other operation you wish to do in the property table. 
+You also must make sure that any references that Schema makes to a table is using the $this->table property.
+   
+In many cases this is simply a case of updating the $table property. In this example we've updated it to:
+
+```
+protected $table = 'products';
+```
+
+The schemas in this example are already pointed to the $this->table property so we don't need to change anything related 
+to the Schema calls.
+
+Migrating an Apartment
+----------------------
+
+Migration, from a user's standpoint, runs exactly the same as a normal laravel application. Simple run:
+
+```
+php artisan migrate
+```
+
+This will apply all the standard Lumen migrations to the public schema ONLY and all the apartment migrations to all the 
+schemas available and NOT to public.
+
+<b>Important:</b> The public database will not contain any of the apartment migrations. If you want the public schema to 
+contain similar migrations to your apartments simple create a standard migration matching your apartments migrations. 
+
+Deleting an Apartment
+---------------------
+You can delete an apartment and its contents by using the command:
+
+```
+php artisan apartment:drop [apartment name]
+```
+
+This will only delete the schema and data for that apartment.
+
+<p align="center">
+
+Within your Lumen Application
+-----------------------------
+
```

Comment:
That was suppose to be centred the same with Command Line Usage with Artisan

It's basically saying How to do it within your application and above how to do it within Artisan, "Command Line Usage with Artisan".

I'm thinking to maybe add ==== under it, as the center hasn't worked.